Resources/doc/transactions.md

```diff
@@ -31,6 +31,26 @@ following example for a hypothetical library bundle.
 class BookController extends FOSRestController
 {
 ```
+
+Transactional class annotation has *writeStatusCodes* config option to copying status codes the transaction to the view (false by default).
```

Comment:
Update to: "Transactional class annotation has a *writeStatusCodes* config option for copying status codes from the transaction to the view (false by default)."